.github/workflows/check.yml

```diff
@@ -25,6 +25,13 @@ jobs:
         with:
```

Comment:
So this all from Rinn's draft PR, right?

How about if we get that merged first, on its own, rather than pulling it into this PR?

Reply:
I merged here to establish that it solved the problem.  I will not push this until Rinn's has been approved and main merged back here.

Reply:
And Rinn's was pushed.  Dismissing review since can't figure out how to resolve.